src/couch_mrview_index.erl

```diff
@@ -230,20 +230,23 @@ update_local_purge_doc(Db, State) ->
 
 verify_index_exists(Options) ->
     ShardDbName = couch_mrview_util:get_value_from_options(<<"dbname">>, Options),
-    IndexName = couch_mrview_util:get_value_from_options(<<"indexname">>, Options),
+    DDocId = couch_mrview_util:get_value_from_options(<<"ddoc_id">>, Options),
     SigInLocal = couch_mrview_util:get_value_from_options(<<"signature">>, Options),
     case couch_db:open_int(ShardDbName, []) of
         {ok, Db} ->
             try
-                DbName =mem3:dbname(Db#db.name),
-                case ddoc_cache:open(DbName, IndexName) of
+                DbName = mem3:dbname(Db#db.name),
+                case ddoc_cache:open(DbName, DDocId) of
                     {ok, DDoc} ->
                         {ok, IdxState} = couch_mrview_util:ddoc_to_mrst(ShardDbName, DDoc),
                         couch_index_util:hexsig(IdxState#mrst.sig) == SigInLocal;
                     _Else ->
                         false
                 end
-            catch _:_ ->
+            catch E:T ->
+                Stack = erlang:get_stacktrace(),
+                couch_log:error("Error occurs when verifying existence of index: ~p ~p",
+                    [{E, T}, Stack]),
```

Comment:
I would change that to ... "verifying existince of ~s/~s :: ~p ~p", [ShardDbName, DDocId, {E, T}, Stack]